zerver/openapi/zulip.yaml

```diff
@@ -14507,6 +14507,50 @@ paths:
         given their user ID.
       parameters:
         - $ref: "#/components/parameters/UserId"
+        - name: is_spammer
+          in: query
+          description: |
+            Whether or not the user should be marked as a spammer.
+
+            **Changes**: New in Zulip 11.0 (feature level ZF-01a225).
+          content:
+            application/json:
+              schema:
+                type: boolean
+              example: true
+          required: false
+        - name: message_delete_action
+          in: query
+          description: |
+            The policy for deleting the user messages after deactivation.
+            An object with boolean values for each message type:
+
+            - `delete_public_stream_messages`: Delete messages in public streams
+            - `delete_private_stream_messages`: Delete messages in private streams
```

Comment:
`Delete messages in private streams => Delete messages in private channels` to follow our policy that we use "channels" in all documentation strings to refer to them.

Also, probably `delete_private_stream_messages` should be `delete_private_channel_messages` to avoid having to rename that later as we move the API towards using "channels" for variable names too.